src/main/java/org/jala/efeeder/places/CreatePlaceCommand.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jala.efeeder.places;
```

Comment:
I think we haven't defined a header for our files yet, so could you please remove these lines?